README.md

```diff
@@ -202,14 +203,13 @@ const href = `/my/other/page?spaceId=${spaceId}&userId=${userId}`
 
 To run OAuth locally, pass the `storyblokApiBaseUrl` property to the params object when calling the `authHandler` function in the target project (i.e. the plugin or application you want to test). With this parameter, you can change the target's backend environment.
 
-
 ```typescript
 import { AuthHandlerParams } from '@storyblok/app-extension-auth'
 
 import { authHandler } from '@storyblok/app-extension-auth'
 const params: AuthHandlerParams = {
   // ...
-  storyblokApiBaseUrl: 'http://localhost:1234'
+  storyblokApiBaseUrl: 'http://localhost:1234',
```

Comment:
We could improve CI to run prettier, so that we don't get these changes (not a problem in this PR)